dev/io.openliberty.microprofile.health.4.0.internal_fat_tck/fat/src/io/openliberty/microprofile/health40/tck/Health40TCKLauncher.java

```diff
@@ -39,7 +40,8 @@ public class Health40TCKLauncher {
     private static final String SERVER_NAME = "Health40TCKServer";
 
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME,
+    public static RepeatTests r = MicroProfileActions.repeatIf(SERVER_NAME,
+                                                             TCKUtilities::areAllFeaturesPresent,
                                                              MicroProfileActions.MP70_EE10,
                                                              MicroProfileActions.MP70_EE11,
                                                              MicroProfileActions.MP61,
```

Comment:
It looks like the refactoring has not formatted these files as the arguments no longer line up.